src/background.js

```diff
@@ -15,6 +15,19 @@ class AccessToken {
     return AccessToken.TOKEN;
   }
 
+  static parse() {
+    if (!AccessToken.TOKEN) {
+      return {};
+    }
+    const base64Url = AccessToken.TOKEN.split('.')[1];
+    const base64 = base64Url.replace(/-/g, '+').replace(/_/g, '/');
+    const jsonPayload = decodeURIComponent(atob(base64).split('').map(function(c) {
+      return '%' + ('00' + c.charCodeAt(0).toString(16)).slice(-2);
+    }).join(''));
```

Comment:
Do you maybe have an example of token? I'm not sure what the format is.

Reply:
this is JWT starndard